app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class QuotesController < ApplicationController
+  def show
+    @quote = Quote.find params[:id]
```

Comment:
`find` throws an error if no record is found.  `find_by` returns nil if no record is found.  Unless I'm misunderstanding something.